src/lib/eliom_state.server.ml

```diff
@@ -94,7 +97,7 @@ let set_global_service_state_timeout
   let sitedata =
     Eliom_request_info.find_sitedata "set_global_service_timeout"
   in
-  let secure = Eliom_common.get_secure secure sitedata in
+  let secure = Eliom_common.get_secure secure sitedata () in
```

Comment:
Yes. We can remove this.
Actually may be we don't want that commit at all.
The important one is the other one, because it points out a very delicate configuration problem ....